src/ConcertoEditor.tsx

```diff
@@ -10,6 +12,34 @@ const concertoTypes = ['String','Integer','Double','DateTime','Long','Boolean']
 
 export default function ConcertoEditor( {value, onChange} : {value: string, onChange?: (value:string|undefined) => void} ) {
 
+    const monacoEditor = monaco.useMonaco();
+    const error = useAppStore((state) => state.error);
+    const ctoErr = useMemo(() => {
+        if (error && error.startsWith("c:")) {
```

Comment:
What does this do? Can't we use the data in the thrown exceptions rather than parsing error strings?

Reply:
This basically filters the concerto related errors from all errors. currently, in redux store, we have one error state that we update on every exception ( whether from template, model or data). 
I could have a separate error state for CTOs if you want, but I was against it because this in the only place where we will be needing it and it looked like all concerto errors will be prefixed by `c:`